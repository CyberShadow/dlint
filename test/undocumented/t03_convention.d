// Ostensibly private by naming convention

void bad1() {}
void _good2() {}
void __good3() {}
