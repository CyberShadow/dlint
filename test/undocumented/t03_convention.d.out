undocumented/t03_convention.d(3,6): Undocumented public declaration: FuncDeclaration `bad1`
